Query log setup (#90)
neo4j/docs-ops-manager: modules/ROOT/pages/addition/index.adoc

----
+
and set environment variables by either setting Environment or EnvironmentFile options.

> For example, using the Environment options, the override file can look as follows:

Reviewer: ```suggestion
For example, using the Environment options, the override file can look like this:
```
